erpnext/hr/report/department_analytics/department_analytics.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2013, Frappe Technologies Pvt. Ltd. and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+import frappe
+from frappe import _
+
+def execute(filters=None):
+	if not filters: filters = {}
+	columns = get_columns()
+	data = get_employees(filters)
+	chart=get_chart_data(data)
+
+	return columns, data, None, chart
+
+def get_columns():
+	return [
+		_("Employee") + ":Link/Employee:120", _("Name") + ":Data:200", _("Date of Birth")+ ":Date:100",
+		_("Branch") + ":Link/Branch:120", _("Department") + ":Link/Department:120",
+		_("Designation") + ":Link/Designation:120", _("Gender") + "::60", _("Company") + ":Link/Company:120"
+	]
+
+def get_employees(filters):
+	conditions = get_conditions(filters)
+	return frappe.db.sql("""select name, employee_name, date_of_birth,
+	branch, department, designation,
+	gender, company from `tabEmployee` where status = 'Active' %s""" % conditions, as_list=1)
+	
+def get_conditions(filters):
+	conditions = ""
+	if filters.get("department"): conditions += " and department = '%s'" % \
+		filters["department"].replace("'", "\\'")
+	return conditions
+
+def get_chart_data(data):
```

Comment:
The chart you implemented does not make much sense to me. Why employees are in X-axis and what are the Y-value means for a employee?
Actually it will be better if show the chart only if data is not filtered by any specific department. And in chart, we can show number of employees based on department. So, on X-axis, there will be department names and in Y-axis number of employees per department.
Also, it will look better in BAR chart.